.travis.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   include:
     - stage: lint
       script: ansible-lint playbooks/*.yml
-    - stage: lint
+    -
       script: yamllint **/*.yml
```

Comment:
Can you have multiple `script` keys for the same `stage`?

Reply:
Yes! It's explained here: https://blog.travis-ci.com/2017-05-11-introducing-build-stages